Resources/Prototypes/Entities/Markers/Spawners/Conditional/timed.yml

```diff
@@ -65,3 +65,26 @@
       minimumEntitiesSpawned: 1
       maximumEntitiesSpawned: 1
 
+- type: entity
+  name: Cockroach Timed Spawner
+  id: CockroachTimedSpawner
+  parent: MarkerBase
+  components:
+    - type: Sprite
+      layers:
+        - state: blue
+        - sprite: Mobs/Animals/cockroach.rsi
+          state: cockroach_icon
+        - state: timed
+    - type: Timer
+    - type: TimedSpawner
+      prototypes:
+        - MobCockroach
+      # round ~90m
+      # one spawner should only spawn ~5 mice for sanity reasons
```

Comment:
if you gonna copy paste mouse spawner why not inherit

Reply:
didnt realise spawners could inherit i'll do this now

Reply:
theyre entities and any entity can have parents :trollface: